### DISCUSSION TITLE
What's the point of "Main menu"

### DISCUSSION THREAD

**Discussion by Seashor** [requarks/wiki#3017] [Help / Questions] [answered]
As a newbee to wikijs, I don't get the point of "Main menu" button.
How can I create pages so that they can be shown under "Main menu" mode ?
Thanks a lot.

**Comment by NGPixel** [comment, marked as answer]
The main menu is a static menu. You define it yourself by adding items to it from the Administration Area > Navigation.

If you don't want a static menu, you can disable it from the same place as above.

**Reply by gihad** [reply]
The "Main Manu" can beautiful (as configured in the wiki.js docs) and perhaps more friendly to end users. It would be great if a representation of it could be auto generated from the existing page structure. I guess that's what the "Site Tree" is for however that feels like you are navigating a file system. 

My concern is that because of it's static nature the Main Menu will not get updated overtime by Wiki editors since it's an extra required step.
